### ISSUE TITLE
Can't run single e2e test per contributing doc

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/libpod#2299]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**
When attempting to run a single test, errors are raised in libpod_suite_test.go

```
# GOPATH=/root/tsweeney/workspaces/podman go test -v test/e2e/libpod_suite_test.go test/e2e/config.go test/e2e/config_amd64.go test/e2e/pull_test.go
# command-line-arguments
test/e2e/libpod_suite_test.go:25:10: undefined: PodmanTestIntegration
test/e2e/libpod_suite_test.go:25:56: undefined: PodmanSessionIntegration
FAIL	command-line-arguments [build failed]
```

**Steps to reproduce the issue:**

See description

**Describe the results you received:**
See description

**Describe the results you expected:**

Test to complete or at least run.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
# podman version
Version:            1.0.1-dev
RemoteAPI Version:  1
Go Version:         go1.10.4
Git Commit:         "afd4d5f4a4b05f421e6f336b4d74a0d808be57ed-dirty"
Built:              Fri Feb  8 17:27:11 2019
OS/Arch:            linux/amd64

```

**Comment by baude** [comment]
add in test/e2e/common.go

**Comment by TomSweeneyRedHat** [comment]
Closed via #2302
